src/runtime/components/forms/Range.vue

```diff
@@ -89,7 +89,13 @@ export default defineComponent({
 
     const { emitFormChange, formGroup } = useFormGroup()
     const color = computed(() => formGroup?.error?.value ? 'red' : props.color)
-    const size = computed(() => formGroup?.size?.value ?? props.size)
+    const size = computed(() => {
```

Comment:
How is this related to the PR?

Reply:
It is the fix in #695. Probably got included by mistake.

Reply:
Would you mind merging the conflicts as well as removing this part?

Reply:
reverted, My bad